### PR TITLE
New driver for OnSemi CAT24AAxx I2C Eeproms.

### PR DESCRIPTION
**Pull request by mikewolfram**

### CODE REVIEW

File: src/modm/driver/storage/cat24aa.hpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+ * I2C Eeprom CAT24AA
+ *
+ * Driver for OnSemi CAT24AA Eeprom.
+ * I2C eeprom with an 16-bit address pointer, encoded in 3 bits of the device
+ * slave address plus the address byte.
+ * Base address is fixed to 0x50.
+ *
+ * @ingroup	modm_driver_i2c_eeprom
+ * @author	Fabian Greif
+ * @author	Niklas Hauser
```
Comment (salkinium) [inline src/modm/driver/storage/cat24aa.hpp]: Update with your name, this is mostly for knowing who to ask for help/bugs/features.

File: src/modm/driver/storage/cat24aa.lb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Niklas Hauser
```
Comment (salkinium) [inline src/modm/driver/storage/cat24aa.lb]: Nope

**Review by salkinium — CHANGES_REQUESTED** [review]
Awesome, just needs more of your name ;-p

File: src/modm/driver/storage/cat24aa.hpp (hunk 1 above)
Comment (mikewolfram) [reply src/modm/driver/storage/cat24aa.hpp]: I should stop working in the hotel bar and start wearing my glasses. :-)

**Review by salkinium — APPROVED** [review]
Thanks!

**Pull request merged**